src/app/views.py

```diff
@@ -1290,13 +1290,40 @@ def archiveRequests(request, license_id=None):
     """ View for archive license requests
     returns archive_requests.html template
     """
+    context_dict = {}
     if request.method == "POST" and request.is_ajax():
-        archive = request.POST.get('archive', False)
-        license_id = request.POST.get('license_id', False)
-        if license_id:
-            LicenseRequest.objects.filter(pk=license_id).update(archive=archive)
+        if request.user.is_authenticated():       
+            user = request.user
+            """ Getting user info for submitting github issue """
+            github_login = user.social_auth.get(provider='github')
+            token = github_login.extra_data["access_token"]
+            username = github_login.extra_data["login"]
+            test = utils.checkPermission(username,token)
```

Comment:
This whole code for checking user perms should be moved to utils...starting from 1296 line.